functions/prepare-check/prepare-entity.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+const azure = require('azure-storage')
+const entGen = azure.TableUtilities.entityGenerator
+const moment = require('moment')
+const R = require('ramda')
+
+/**
+ * Return an entity suitable for inserting into the `preparedCheck` table
+ * @param preparedCheck
+ * @return {{PartitionKey: azurestorage.services.table.TableUtilities.entityGenerator.EntityProperty<string>, RowKey: azurestorage.services.table.TableUtilities.entityGenerator.EntityProperty<string>, checkCode: azurestorage.services.table.TableUtilities.entityGenerator.EntityProperty<any>, collectedAt: null, config: azurestorage.services.table.TableUtilities.entityGenerator.EntityProperty<string>, createdAt: azurestorage.services.table.TableUtilities.entityGenerator.EntityProperty<Date>, isCollected: azurestorage.services.table.TableUtilities.entityGenerator.EntityProperty<boolean>, pinExpiresAt: azurestorage.services.table.TableUtilities.entityGenerator.EntityProperty<Date>, pupil: azurestorage.services.table.TableUtilities.entityGenerator.EntityProperty<string>, pupilId: azurestorage.services.table.TableUtilities.entityGenerator.EntityProperty<number>, questions: azurestorage.services.table.TableUtilities.entityGenerator.EntityProperty<string>, school: azurestorage.services.table.TableUtilities.entityGenerator.EntityProperty<string>, schoolId: azurestorage.services.table.TableUtilities.entityGenerator.EntityProperty<number>, tokens: azurestorage.services.table.TableUtilities.entityGenerator.EntityProperty<string>, updatedAt: azurestorage.services.table.TableUtilities.entityGenerator.EntityProperty<Date>}}
+ */
```

Comment:
Urgh!

Reply:
I'm guessing that this is auto-generated.

It's azure... if you want to eat Rhino you have to be prepared to pay freight as someone said